Classes/Console/Command/Upgrade/UpgradeAllCommand.php

```diff
@@ -40,7 +40,7 @@ protected function createNativeDefinition(): array
             new InputOption(
                 'arguments',
                 'a',
-                InputOption::VALUE_REQUIRED,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
```

Comment:
Instead this should be added after line 67
```php
        if (!is_array($arguments)) {
            $arguments = explode(',', $arguments);
        }
```

Reply:
Have added the check inline like other commands like InstallGeneratePackageStatesCommand.